Comments for changes

oj-kentd opened the pull request:
Putting comments in line with changes to reflect better units and cleaner example.

BryceVichiOJ commented:
Makes sense to me. I'll make these changes and update this PR.

BryceVichiOJ commented:
All requested changes were put in place. Clear to begin review process.

nichmj01 commented on CMakeLists.txt:
```diff
@@ -1,14 +1,14 @@
 
 ## Generate added messages and services
-generate_messages(DEPENDENCIES std_msgs)
+#generate_messages(DEPENDENCIES std_msgs)
```
Why was this commented out?

BryceVichiOJ replied:
std_msgs is declared as a dependency to the ROS project. This line in CMakeLists and the dependency in ROS have the same effect. Therefore, it is not needed. Why I chose to comment it instead of removing, I'm not sure. I'll go in and remove the comment.

BryceVichiOJ replied:
Comment removed.

BryceVichiOJ committed:
Add Units to Variable and Function Names

- Added units to certain variables and functions
- Change CMake to accept all version of Parakeet SDK rather than specifically 1.0.0
- Make use of std::min and std::max when finding the min/max of ranges and angles
